File: Python/1000-1999/1133.py
def main():
    temp = int(input())
    temp2 = int(input())

    ini = min(temp, temp2)
    fim = max(temp, temp2)

    for c in range(ini+1, fim):
        if c%5 == 2 or c%5 == 3:
            print(c)

main()

File: C/1000-1999/1133.c
#include <stdio.h>

int main (void){
    int x, y, temp;

    scanf("%d", &x);
    scanf("%d", &y);

    if (x > y){
        temp = x;
        x = y;
        y = temp;
    }

    for (x= x+1; x<y; x++){
        if (x%5 == 2 || x%5 == 3){
            printf("%d\n", x);
        }
    }

    return 0;
}
